tests/requirements.txt

```diff
@@ -11,8 +11,7 @@ uvloop; platform_system != "Windows"
 
 bpython  # optional for shell wrapper tests
 brotli; implementation_name != 'pypy'  # optional for HTTP compress downloader middleware tests
-# 1.1.0 is broken on PyPy: https://github.com/google/brotli/issues/1072
-brotli==1.0.9; implementation_name == 'pypy'  # optional for HTTP compress downloader middleware tests
+brotlicffi; implementation_name == 'pypy'  # optional for HTTP compress downloader middleware tests
 zstandard; implementation_name != 'pypy'  # optional for HTTP compress downloader middleware tests
```

Comment:
Oh, so brotli and zstandard are here. It would be great if we could remove them from here and put them in the extra environments of `tox.ini` instead, to make sure Scrapy works without them. (this also applies to other packages here, but they are out of scope; in fact, `zstandard` is kind of out-of-scope, you could leave it here as well).